app/assets/javascripts/movies.js

```diff
@@ -0,0 +1,19 @@
+$(function() {
+  $('html').on('change', 'select#movie_event_ids.chosen', function(){
+    var form = $(this).parent('form');
+
+    $.ajax({
+      type: "GET",
+      url: form.attr('action') + "/new",
+      success: function(data){
+        var chosenEventID = $('#movie_event_ids').val();
+        var chosenEvent = data.find(function(event){ return event["id"] == chosenEventID; });
+        var chosenEventImageURL = chosenEvent["first_image_url"];
+        $('#movie_poster_url').val(chosenEventImageURL);
+      },
```

Comment:
Often, `get` ajax call return a piece of html that you could use to change a piece of your DOM... Could it be a better idea here ?